manager/cve_handler.py

```diff
@@ -20,6 +21,12 @@
 
 LOGGER = get_logger(__name__)
 
+PATCH_HOST = os.getenv('PATCH_HOST', 'http://patchman-engine-manager.patchman-engine-ci.svc:8080')
+PATCH_ADVISORIES_API = os.getenv("PATCH_ADVISORIES_API", "/api/patch/v1/advisories")
+PATCH_FILTER_QUERY = os.getenv('PATCH_FILTER_QUERY', '?filter[id]=in:')
```

Comment:
PATCH_ADVISORIES_API and PATCH_FILTER_QUERY are variables but are not in manager.env config file

either:
1. put them in the manager.env file
2. or add them to "not_configurable=" in run_tests.sh